CalibCalorimetry/EcalLaserAnalyzer/interface/TFParams.h

```diff
@@ -38,11 +35,14 @@ matrice cree_mat_prod(matrice, matrice);
 void fill_mat(matrice, matrice);
 matrice fill_mat_int(matrice, matrice, matrice);
 
-#define dimmat 30
-#define dimout 10
-#define nbmax_cell 1000
-
 class TFParams : public TObject {
+public:
```

Comment:
@davidlange6 just for my understanding: is there a specific reason why you define the `static constexpr` in this "new" public instead of adding them in [line 59](https://github.com/cms-sw/cmssw/pull/33320/files#diff-ab8579c2e46c1489e75b39b938c7ebc814944796d00007527dd086b4283a46baR59) ?

Reply:
Not particularly, no. public interfaces tend to be at the beginning of a class (eg, https://google.github.io/styleguide/cppguide.html#Declaration_Order)